### PR TITLE
Feature/lookup with ethr did registry

### PR DESCRIPTION
**Pull request by oed**

### CODE REVIEW

File: src/lookup.js [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
-  )).result
-  return hexHash === ZERO_HASH ? null : hexToIpfsHash(hexHash)
+async function ethrLookup (managementKey, rpcUrl = PROVIDER_URL) {
+  const eth = new Eth(new HttpProvider(rpcUrl))
+  const logDecoder = abi.logDecoder(EthrDIDRegistry.abi, false)
+  const address = managementKey.length === 42 ? managementKey.slice(2).toLowerCase() : toEthereumAddress(managementKey)
+  const logs = logDecoder(await eth.getLogs({
+    address: EthrDIDRegistry.address,
+    topics: [ATTRIBUTE_CHANGED_FILTER, '0x000000000000000000000000' + address],
+    fromBlock: 0,
```
Comment (pelle) [inline src/lookup.js]: @oed this is going to be very slow. Have a look at the approach I used in ethr-did-resolver

Comment (oed) [reply src/lookup.js]: Tnx, will make the change!

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "muport-did-resolver",
-  "version": "0.1.3",
+  "version": "0.2.0-beta-4",
```
Comment (christianlundkvist) [inline package.json]: Why the `beta-4` stuff?

File: package.json [hunk 3]
```diff
@@ -32,19 +32,21 @@
...
     "babel-preset-env": "^1.6.0",
     "bluebird": "^3.5.1",
-    "ethereum-claims-registry": "^0.0.4",
     "ganache-cli": "^6.1.0",
     "jest": "^22.4.0",
     "regenerator-runtime": "^0.11.1",
     "sinon": "^4.0.1",
     "standard": "^11.0.0",
-    "web3": "0.20.6"
+    "web3": "^1.0.0-beta.34"
```
Comment (christianlundkvist) [inline package.json]: Why use the beta of web3 1.0.0?

Comment (oed) [reply package.json]: It's the default when you install web3. The `0.2` is not supported anymore and throws error when you try to install it.

File: package.json (hunk 2 above)
Comment (oed) [reply package.json]: I needed to release a beta version so I could use it in the muport-core library when testing. Will make a release when this is merged.

File: package.json (hunk 3 above)
Comment (christianlundkvist) [reply package.json]: Ok got it. Weird that they call it 1.0 beta when 0.2 is no longer supported :\

File: package.json (hunk 2 above)
Comment (christianlundkvist) [reply package.json]: You'll make another PR removing the beta?

Comment (oed) [reply package.json]: I'll just do it in master. We don't have a develop branch yet, but if development on this repo get's more active I'll add one. For now I feel it's just a waste of time :)

**Pull request merged**